### PR TITLE
BaSP-TG-40: fix crash when deleting an employee in project

### PR DESCRIPTION
**Pull request by AlejoFGM**

### CODE REVIEW

File: src/Components/Projects/index.js [hunk 1]
```diff
@@ -71,7 +71,11 @@ const Projects = () => {
                 {project.employees.map((employee) => {
                   return (
                     <tr key={employee._id}>
-                      <td>{employee.employee.name}</td>
+                      <td>
+                        {employee.employee === null
```
Comment (Eju97) [inline src/Components/Projects/index.js:75]: You can also use `{!employee.employee ? 'There is no employee' : employee.employee.name} `

File: src/Components/Projects/Form/index.js [hunk 2]
```diff
@@ -46,19 +46,22 @@ const Form = () => {
         });
         const project = await response.json();
         const employeeList = project.data.employees.map((item) => {
-          return {
-            employee: item.employee._id,
-            rate: item.rate,
-            role: item.role
-          };
+          if (item.employee !== null) {
```
Comment (Eju97) [inline src/Components/Projects/Form/index.js:49]: Same here! You can use `(item.employee)` if I'm not wrong.

File: src/Components/Projects/Form/index.js [hunk 3]
```diff
@@ -46,19 +46,22 @@ const Form = () => {
...
-            rate: item.rate,
-            role: item.role
-          };
+          if (item.employee) {
+            return {
+              employee: item.employee._id,
+              rate: item.rate,
+              role: item.role
+            };
+          } else return 'noEmployee';
```
Comment (lmalvasia) [inline src/Components/Projects/Form/index.js:55]:
```suggestion
          }
          return 'noEmployee';
```

**Review by lmalvasia — APPROVED** [review]

**Comment by Tibor700** [comment]
all entities works very well excepet timesheet that it crashed suddently

**Pull request merged**